doc/Web_Runtime_Test_Suite_Developer_Guide.md

```diff
@@ -2,310 +2,76 @@
-
-To contribute new cases to test suite package, perform the following steps:
-
-1)Design new test case according to WRT Spec and add new case information to **tests.xml**. For details, see "Appendix 1 Tests.full.xml and tests.xml."
-
-- "Case name" should follow the test case naming convention. For details, see chapter 4 "Test Case Naming Convention".
-- "Specs" field should follow the Spec coverage assertion rules. For details, see chapter **Error! Reference source not found.** "Test Case Classification (<testcase\> field in tests.xml)".
-- "Component" field should comply with the WRT component name list.
+Note:
 
-2)Develop test script by following the test case coding style and put it under <testcasefolder\>.
+- Each test should have an entry HTML file/Python script.
+- The test script can be embedded into the HTML/Python file or be used as separated JavaScript/Python file.
+- Set `<set type='pyunit'>` in `tests*.xml` for PyUnit test.
 
-Note:
 
-- Each test should have an entry HTML file.
-- The test script can be embedded into the HTML file or be used as separate JavaScript file.
-- For details on how to use W3C test harness, see [testharness.js API document](https://github.com/w3c/testharness.js/blob/master/docs/api.md).
-
-**Example**
-
-    <meta charset='utf-8'>
-    <title>Crosswalk_I18n_AppNameChinese_Test</title>
-    <link rel="author" title="Intel" href="http://www.intel.com">
-    <p>
-      <strong>Test steps:</strong>
-    </p>
-    <ol>
-      <li>install i18n_locales_chinese_tests wgt app </li>
-      <li>Change the System Settings language to Chinese</li>
-      <li>Check the web app name and launcher it </li>
-    </ol>
-    <p>
-      <strong>Expected Output:</strong>
-    </p>
-    <ol>
-      <li>The app can be install successfully</li>
-      <li>The device language is set to Chinese successfully</li>
-      <li>Web app name with Chinese and launch successfuly</li>
-    </ol>
-
-## 9 How to Contribute pairwise Cases to Test Suite Package
+## 4 How to contribute pairwise cases for Web Runtime?
 
 - Install  AllPairs
 
-  - Download AllPairs from here:
-
-     http://sourceforge.net/projects/allpairs/
+1. Download AllPairs from here:
 
-  - Copy metacomm folder to your suite:
-     metacomm/
+   http://sourceforge.net/projects/allpairs/
 
-  - import metacomm in your python file:
+2. Copy metacomm folder to your suite:
+   metacomm/
 
-    import metacomm.combinatorics.all_pairs2
+3. import metacomm in your python file:
 
-  -  all_pairs sample:
+  import metacomm.combinatorics.all_pairs2
 
-     list=[[1,2],[3]]
-     input_pair = all_pairs( list )
-     for e, v in enumerate(input_pair):
-         print e,v
+4. all_pairs sample:
 
-....0 [1, 3]
-....1 [2, 3]
+       ```
+        list=[[1,2],[3]]
+        input_pair = all_pairs( list )
+        for e, v in enumerate(input_pair):
+            print e,v
 
+        ....0 [1, 3]
+        ....1 [2, 3]
+       ```
 
 ## Appendix 1 `tests.full.xml` and `tests.xml`
```

Comment:
This item has been introduced in ./Web_Test_Suite_Developer_Guide.md, pls. remove it.